custom_components/daikinskyport/daikinskyport.py

```diff
@@ -143,40 +73,40 @@ def get_thermostats(self):
             return self.thermostats
         else:
             self.authenticated = False
-            logger.debug("Error connecting to Daikin Skyport while attempting to get "
+            _LOGGER.debug("Error connecting to Daikin Skyport while attempting to get "
                         "thermostat data.  Refreshing tokens and trying again.")
             if self.refresh_tokens():
                 return self.get_thermostats()
             else:
                 return None
 
-    def get_thermostat_info(self, deviceid):
+    async def get_thermostat_info(self, deviceid):
         ''' Retrieve the device info for the specific device '''
         url = 'https://api.daikinskyport.com/deviceData/' + deviceid
         header = {'Content-Type': 'application/json;charset=UTF-8',
                   'Authorization': 'Bearer ' + self.access_token}
         try:
             request = requests.get(url, headers=header)
         except RequestException:
-            logger.warn("Error connecting to Daikin Skyport.  Possible connectivity outage.")
+            _LOGGER.warn("Error connecting to Daikin Skyport.  Possible connectivity outage.")
             return None
         if request.status_code == requests.codes.ok:
             self.authenticated = True
             return request.json()
         else:
             self.authenticated = False
-            logger.debug("Error connecting to Daikin Skyport while attempting to get "
+            _LOGGER.debug("Error connecting to Daikin Skyport while attempting to get "
                         "thermostat data.  Refreshing tokens and trying again.")
             if self.refresh_tokens():
                 return self.get_thermostat_info(deviceid)
             else:
                 return None
 
-    def get_thermostat(self, index):
+    async def get_thermostat(self, index):
```

Comment:
Same thoughts here as I said with logging above: "Why?" and "should this be split into a separate PR?"